### PR TITLE
[bitnami/metrics-server] Fix wrong template helper usage

### PR DESCRIPTION
**Pull request by rocketspacer**
**Description of the change**

#4529 introduced wrong usages of template helpers. This PR fixes them.

**Benefits**

Fix broken template

**Possible drawbacks**

N/A

**Applicable issues**

N/A

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/metrics-server/templates/pdb.yaml [hunk 1]
```diff
@@ -2,14 +2,8 @@
...
 metadata:
-  labels:
-    app: {{ template "metrics-server.name" . }}
-    chart: {{ template "metrics-server.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-  name: {{ template "metrics-server.fullname" . }}
-  namespace: {{ .Release.Namespace }}
-
+  name: {{ template "common.names.fullname" . }}
```
Comment (dani8art) [inline bitnami/metrics-server/templates/pdb.yaml:5]: Could we use `include`?

```suggestion
  name: {{ include "common.names.fullname" . }}
```

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi thanks for the PR, please take a look at my comment.

File: bitnami/metrics-server/templates/pdb.yaml (hunk 1 above)
Comment (rocketspacer) [reply bitnami/metrics-server/templates/pdb.yaml]: Fixed

**Review by dani8art — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**